README.md

```diff
@@ -1,57 +1,348 @@
+Is an application framework for simulating a proof-of-concept environment that demonstrates an approach to gaining transparency on the environmental costs of network paths and optimising the flow of data within that network to reduce the carbon footprint caused by its operation.
 
-## Overall Project Achievements
+**Currently** it contains the following components: 
 
-### Before Hackathon
+- **Network** based on BMv2 software switches capable of:
+  - Collecting inband network telemetry (INT) data regarding energy efficiency of network paths using the IOAM protocol
+  - Exporting the INT data using IPFIX
+- **Configuration** utilities capable of:
+  - Provisioning of an arbitrary network topology with information out of a declarative YAML definition
+  - Dynamic configuration update of network paths and network efficiency which uses the [nornir](https://github.com/nornir-automation/nornir) network automation framework
+- **Monitoring** system based on the Telegraf, InfluxDB, Grafana (TIG) Stack capable of:
+  - Collection  of IPFIX messages ([Telegraf](https://www.influxdata.com/time-series-platform/telegraf/))
+  - Parsing of IOAM raw export IPFIX messages ([Telegraf](https://www.influxdata.com/time-series-platform/telegraf/))
+  - Persistent storage of efficiency data ([InfluxDB](https://www.influxdata.com/))
+  - Visualization of efficiency data ([Grafana](https://grafana.com/))
 
-- Dynamic Grafana Dashboards to allow simulations on arbitrary topologies
-- Skeleton of green-path-optimizer application
-  - Retrieve data from InfluxDB
-- Implementation of flooding mechanism for probing packets
-  - Loops? / Spanning Tree
+**In near futre** it will also contain the following components:
 
-### During Hackathon
-- Analyse given raw data in InfluxDB
-- **Write green-path-optimizer application which suggests path updates to improve efficiency**
-- **Connect green-path-optimizer with the configuration-update-utility to acutally deploy the improvements in the PoC**
-- In case there is time:
-  - Parameter evaluation (Maximum Power Draw)
-  - Interpretation of improvements
-  - Validation of the suggested improvements also regarding throughput / bottlenecks etc.
+- **Optimizer** to be developed at [RIPE NCC Green Tech Hackathon](https://labs.ripe.net/author/becha/announcing-the-green-tech-hackathon/) which is capable of:
+  - Analyzing the given time series data stored in InfluxDB
+  - Proposing configuration changes to route traffic over the most efficient paths between every possible ingress and egress router
+  - Trigger configuration update to actually implement the proposed optimization in the network.
+- **Validator** which is to be developed after the hackathon and validates the optimiser's configuration changes to e.g. prevent data from being sent through bottlenecks or certain paths from being overprovisioned.
 
-### After Hackathon
+## Contents
```

Comment:
I would probably start with the content section. The part above could be within an Introduction section. So, the reader can jump to the appropriate chapter directly.